lamarr_hybrid/roster.json

```diff
@@ -37,7 +37,13 @@
       },
       {
         "name": "Mark Halstead",
-        "github": "Mark-Halstead"
+        "github": "Mark-Halstead",
+        "materials": [
+        {
+          "type": "video",
+          "url": "https://youtu.be/oQ4atw_mpko"
+        }
```

Comment:
Can you please update to 
 {
     "type": "video",
     "url": "oQ4atw_mpko"
  }